tests/playbooks/tests_ethtool_ring.yml

```diff
@@ -0,0 +1,102 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    interface: testnic1
+    type: veth
+  tasks:
+    - debug:
+        msg: "this is: playbooks/tests_ethtool_ring.yml"
+      tags:
+        - always
+
+    - name: "INIT: Ethtool ring tests"
+      debug:
+        msg: "##################################################"
+    - include_tasks: tasks/show_interfaces.yml
+    - include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - include_tasks: tasks/assert_device_present.yml
+    - name: Install ethtool (test dependency)
+      package:
+        name: ethtool
+        state: present
+
+    - block:
+        - name: >-
+            TEST: I can create a profile without any ring option.
+          debug:
+            msg: "##################################################"
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                type: ethernet
+                autoconnect: no
+                ip:
+                  dhcp4: no
+                  auto6: no
+        - name: Get profile's ring options
+          command: nmcli -g ethtool.ring-rx c show {{ interface }}
+          register: no_ring
+        - name: "ASSERT: The profile does not contain ring options"
+          assert:
+            that: no_ring.stdout | length == 0
+        - name: >-
+            TEST: I can set rx.
```

Comment:
If possible, it would be great to set all four settings.

Reply:
Four settings have been set.